src/app/user/booking/new/page.tsx

```
@@ -518,8 +512,9 @@ function mockAvailableSlots(agentId: string, date: string, baseSlots: string[]):
 }
 
 // MAIN PAGE COMPONENT
-export default function NewBookingPage() {
+function NewBookingPage() {
     const router = useRouter();
+    const searchParams = useSearchParams();
     const { user, isAuthenticated, isLoading } = useAuth();
     const [searchParams, setSearchParams] = useState<URLSearchParams | null>(null);
```

Biome (2.1.2)

[error] Shouldn't redeclare 'searchParams'. Consider to delete it or rename it.

'searchParams' is defined here:


(lint/suspicious/noRedeclare)